AEA-795 Skills Readme (#1735)
fetchai/agents-aea: packages/fetchai/skills/aries_alice/README.md

# Aries Alice

## Description

This skill emulates the Alice actor in <a href="https://github.com/hyperledger/aries-cloudagent-python/blob/master/demo/README.md" target="_blank">this demo</a>.

> This skill is part of the Fetch.ai Aries demo. It simulates the Alice actor of the demo linked above. It first registers Alice ont he soef. It then receives invitation details from Faber AEA. Then it connects with an underlying aries cloud agent (ACA) instance and executes an `accept-invitation` command.

Reviewer: ```suggestion
This skill is part of the Fetch.ai Aries demo. It simulates the Alice actor of the demo linked above. It first registers Alice on the SOEF. It then receives invitation details from Faber AEA. Then it connects with an underlying aries cloud agent (ACA) instance and executes an `accept-invitation` command.
```
